Tests/PointFreeTests/EpisodeTests.swift

```diff
@@ -16,10 +16,21 @@ class EpisodeTests: TestCase {
   override func setUp() {
     super.setUp()
     update(&Current, \.database .~ .mock)
+//    record=true
   }
 
   func testEpisodePage() {
-    let episode = request(to: .episode(.left(Current.episodes().first!.slug)), session: .loggedOut)
+    let episodes: [Episode] = [
+      .mock |> \.id .~ 1 |> \.sequence .~ 1 |> \.title .~ "Functions Part 1",
+      .mock |> \.id .~ 2 |> \.sequence .~ 2 |> \.title .~ "Functions Part 2",
+      .mock |> \.id .~ 3 |> \.sequence .~ 3 |> \.title .~ "Functions Part 3",
```

Comment:
might be good to have test coverage with there's no next episode and no previous episode too.